packages/calypso-products/src/plans-list.js

```diff
@@ -900,6 +906,8 @@ const getPlanJetpackSecurityT2Details = () => ( {
 	getHiddenFeatures: () => [
 		FEATURE_JETPACK_BACKUP_REALTIME,
 		FEATURE_JETPACK_BACKUP_REALTIME_MONTHLY,
```

Comment:
These two items in the hidden features array cause Backup Real-time to show up at the bottom of the product grid whenever someone owns an active subscription to Security Tier 2. If there aren't any unwanted side effects, I think we should remove them. (This also applies to Security Tier 1, on lines 865-866.)

```suggestion
```